example-local.yaml

```diff
@@ -3,14 +3,41 @@ packages:
   econet: !include
     # Change this file to the correct one for your device
     file: econet_heatpump_water_heater.yaml
+    # next line is needed for all hvac single zone installs
     # file: econet_hvac.yaml
+    # next line is needed for all hvac multi zone installs (do not include above)
+    # file: econet_hvac_base.yaml
     # Use this if you have a communicating outdoor unit (ODU)
     # files: [econet_hvac.yaml, econet_hvac_odu.yaml]
     # file: econet_tankless_water_heater.yaml
     # file: econet_electric_tank_water_heater.yaml
     vars:
       external_components_source: components
 
+  # for zones - include the econet hvac package above and the appropriate zones below
```

Comment:
I think it might be better to revert changes to this file and create a new one: example-hvac-zones.yaml with 3 zones and a comment to adjust (add or remove). It will be easier for users.

Reply:
agreed